backend/asmgen.ml

```diff
@@ -142,12 +147,17 @@ let compile_fundecl ~ppf_dump fd_cmm =
   ++ Profile.record ~accumulate:true "linearize" Linearize.fundecl
   ++ pass_dump_linear_if ppf_dump dump_linear "Linearized code"
   ++ (fun (fd : Linear.fundecl) ->
-      if !use_ocamlcfg then begin
-        let cfg = Linear_to_cfg.run fd ~preserve_orig_labels:true in
+    if !use_ocamlcfg then begin
+      fd
+      ++ Profile.record ~accumulate:true "linear_to_cfg"
+           (Linear_to_cfg.run ~preserve_orig_labels:false)
```

Comment:
Not sure it matters, but `preserve_orig_labels` was `true` before.

Reply:
fixed, thanks.